backend/lib/endpoints/users.js

```diff
@@ -37,6 +37,17 @@ async function routes(app) {
       };
     },
   );
+
+  app.put(
+    "/signup/createProfile",
+    { schema: createProfileSchema },
```

Comment:
Relating to my previous comment about using PUT, if we're updating an existing user profile I think we should add a authentication prevalidation so that only logged in users can update their profile (if that's the intention of this endpoint). We would also need to make sure the currentUser is the owner of the profile if that's the case.

Reply:
yeah definitely needs authentication first and extra check that you can only update your own user data, but see previous comments about which HTTP method and endpoint to use